packages/devtools_app/lib/src/inspector/flutter/inspector_screen_details_tab.dart

```diff
@@ -0,0 +1,229 @@
+    ];
+    final tabViews = <Widget>[
+      detailsTree,
+      if (enableExperimentalStoryOfLayout)
+        LayoutDetailsTab(controller: summaryTreeController),
+    ];
+    final focusColor = Theme.of(context).focusColor;
+    return Container(
+      decoration: BoxDecoration(
+        border: Border.all(color: focusColor),
+      ),
+      child: DefaultTabController(
+        length: tabs.length,
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: <Widget>[
+            Container(
+              padding: const EdgeInsets.only(bottom: 8.0),
+              child: Row(
+                children: <Widget>[
+                  Container(
+                    color: Theme.of(context).focusColor,
+                    child: TabBar(
+                      tabs: tabs,
+                      isScrollable: true,
+                    ),
+                  ),
+                  if (expandCollapseSupported)
+                    Expanded(
+                      child: expandCollapseButtons,
+                    ),
+                ],
+                mainAxisAlignment: MainAxisAlignment.spaceBetween,
+              ),
+            ),
+            Expanded(
+              child: TabBarView(
+                children: tabViews,
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class LayoutDetailsTab extends StatefulWidget {
+  const LayoutDetailsTab({Key key, this.controller}) : super(key: key);
+
+  final InspectorTreeControllerFlutter controller;
+
+  @override
+  _LayoutDetailsTabState createState() => _LayoutDetailsTabState();
+}
+
+class _LayoutDetailsTabState extends State<LayoutDetailsTab>
+    with AutomaticKeepAliveClientMixin<LayoutDetailsTab>
+    implements InspectorControllerClient {
+  InspectorTreeControllerFlutter get controller => widget.controller;
+
+  RemoteDiagnosticsNode get selected => controller.selection?.diagnostic;
+
+  // Lifecycle hooks
+  @override
+  void initState() {
+    super.initState();
+    controller.addClient(this);
+  }
+
+  @override
+  void dispose() {
+    super.dispose();
+    controller.removeClient(this);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    super.build(context);
+    if (selected == null)
+      return const Center(
+        child: CircularProgressIndicator(),
+      );
+    if (!selected.isFlex)
+      return Container(
+        // TODO(albertusangga): Visualize non flex widget constraint with Bounding Box model
+        child: const Text('TODOs for Non Flex widget'),
+      );
+    return StoryOfYourFlexWidget(
+      diagnostic: selected,
+      // TODO(albertusangga): Optimize and cache deserialization instead of always calling it when state changes
+      properties: RenderFlexProperties.fromJson(selected.renderObject),
+    );
+  }
+
+  @override
+  bool get wantKeepAlive => true;
+
+  @override
+  void onChanged() {
+    setState(() {});
```

Comment:
this is also going to get called at the wrong times as we don't need to update just because the inspector tree needs to update/

Reply:
I thought this method is only called when `onSelectionChanged` is triggered, seems not to be the case?

Reply:
removed this in the latest diff